tech/database/cassandra/cassandra_documentation.md

```diff
@@ -0,0 +1,7 @@
+## Documentation
```

Comment:
Should be probably in a "about.md" page.

Also it's missing a header (I think for most of the pages, along with the `##` instead of `#`).